openbook_auth/models.py

```diff
@@ -2056,6 +2063,13 @@ def _check_follow_lists_ids(self, lists_ids):
     def _check_follow_list_id(self, list_id):
         self._check_has_list_with_id(list_id)
 
+    def _check_can_edit_post_with_id(self, post_id):
```

Comment:
The operation is called `update_post_with_id`, therefore this should be `_check_can_update_post_with_id`